wynnset/tapestry-wp
# 753 - Fix and Improve WP Init Script

Addresses: #753.
When merged, this PR should

- [x] ensure that the docker Wordpress Initialization script is able to find the wp installation and perform the install within a reasonable amount of time.
- [x] add a way to programmatically put API key in for config.js for LinkPreview

**Testing Instructions**:

0. Make sure Docker daemon is running.

1. In root project, run `make uninstall`. This will remove all data related to the WordPress setup, including pages and  users!

2. Run `make init`. This should load up the wordpress containers and then run the installation script. Keep an eye on the output until you see a message that WP was successfully installed. You should now be prompted to enter your LinkPreview key.

3. If it worked properly, run `make start-app` to start the tapestry application. Now, open up the application in a browser and ensure that external links are working properly (create an external link node with a working thumbnail such as a Wikipedia entry) , otherwise run `make stop` to bring down all containers and try increasing the container delay as described in the file`bin/config-sample.sh` (and run `make-init` again).
